packages/client/src/v2-events/features/workqueues/Workqueue.tsx

```diff
@@ -241,7 +242,7 @@ function Workqueue({
 
   function getDefaultColumns(): Array<Column> {
     return workqueueConfig.defaultColumns.map(
-      (column): Column => ({
+      (column: string): Column => ({
```

Comment:
In my opinion its better to leave these types inferred, explicitly typing won't improve the type here

<img width="370" alt="Screenshot 2025-05-02 at 11 05 35" src="https://github.com/user-attachments/assets/d8057043-6cab-43f7-aaca-a297e4595a99" />

This one saves one import as well:

<img width="625" alt="Screenshot 2025-05-02 at 11 06 10" src="https://github.com/user-attachments/assets/32a16332-5b74-475d-8182-5ea79cdfdbbf" />

But this is a very minor nitpick thing, either way is ok :)

Reply:
it was showing as an issue in linter